packages/example-log-extension/README.md

```diff
@@ -109,8 +110,16 @@ export enum LOG_LEVEL {
 }
 ```
 
+
 ### `src/types.ts`
-Define TypeScript type definitions / interfaces for complex types and functions here.
+Before we continue, we will need to install a new dependecy as follows:
+
+```shell
+npm i @loopback/rest
```

Comment:
`--save` or `--save-dev` here?

Reply:
`--save` is the default behaviour with npm 5